nodewatcher/modules/defaults/network_profile/defaults/wireless.py

```diff
@@ -163,6 +163,21 @@ def get_project_ssid(purpose, attribute='essid'):
                         'routing_protocols': routing_protocols,
                     },
                 )
+
+            # Create 802.11s mesh interface.
+            if 'no-wifi-mesh' and '802.11s-wifi-mesh' in network_profiles:
+                self.setup_interface(
+                    state,
+                    cgm_models.WifiInterfaceConfig,
+                    parent=wifi_radio,
+                    configuration={
+                        'mode': '802.11s',
+                        'essid': get_project_ssid('802.11s'),
+                        'mesh_key': get_project_ssid('802.11s_mesh_id'),
```

Comment:
This is probably not ok as the model doesn't have a `mesh_key` field. Why don't you just set `bssid` and then use `bssid` as `mesh_id` when configuring UCI in CGMs?